src/auspex/filters/correlator.py

```diff
@@ -19,17 +19,8 @@ class Correlator(ElementwiseFilter):
     def operation(self):
         return np.multiply
 
-    def update_descriptors(self):
-        logger.debug('Updating correlator "%s" descriptors based on input descriptor: %s.', self.name, self.sink.descriptor)
+    def filter_name(self):
```

Comment:
This is specific to elementwise filters, which are supposed to have dirt-simple subclasses, but I've moved to a cleaner way of doing things.